src/utils/command-helpers.mjs

```diff
@@ -174,12 +174,18 @@ export const warn = (message = '') => {
 
 /**
  * throws an error or log it
- * @param {string|Error} message
+ * @param {unknown} message
  * @param {object} [options]
  * @param {boolean} [options.exit]
  */
 export const error = (message = '', options = {}) => {
-  const err = message instanceof Error ? message : new Error(message)
+  const err =
```

Comment:
Just like the other error change, I'd love some more context here.

Reply:
yea often this is used to just pipe the result from a try catch in it.

like:
```javascript
try {
 // some code
} catch (_error)
  error(_error)
}
```
It's not guaranteed that the message is always a string it can be an error class or you can basically throw any number object or whatever. 

By introducing type guards here and making the `message` of type `unknown` we are safer on all eventualities